internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -879,6 +880,106 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				if err != nil {
 					log.Fatal(err)
 				}
+			case "cloudflare_teams_list":
+				jsonPayload, _, err := api.ListTeamsLists(context.Background(), &cloudflare.ResourceContainer{Identifier: accountID}, cloudflare.ListTeamListsParams{})
```

Comment:
we already conditionally build the resource identifier for reuse

```suggestion
				jsonPayload, _, err := api.ListTeamsLists(context.Background(), identifier, cloudflare.ListTeamListsParams{})
```